### DISCUSSION TITLE
为什么老是有人发色图啊啊啊啊

### DISCUSSION THREAD

**Discussion by Boring-Coils** [share121/inter-knot#2996] [其他]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by ybbgh** [comment]
❤点❤我❤观❤看❤邦❤布❤可❤爱❤录❤像❤

**Reply by lllliufeng** [reply]
是真的

**Reply by Boring-Coils** [reply]
啊？

**Reply by ybbgh** [reply]
<a href="https://www.bilibili.com/video/BV17e8ceeEGq/?spm_id_from=333.788.recommend_more_video.5&vd_source=9f1c67940ef7fee251eca4e9f4089507" target="_blank">❤点❤我❤观❤看❤邦❤布❤可❤爱❤录❤像❤</a>

**Reply by weilai0001** [reply]
还真能点啊你

**Comment by AAA-0012** [comment]
因为人的xp是自由的

**Comment by Azazel-001** [comment]
发我看看

**Comment by guanyq000** [comment]
怎么会有这种人！链接发我！我要去严惩！
